x-pack/plugins/enterprise_search/server/routes/app_search/engines.test.ts

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { RequestHandlerContext } from 'kibana/server';
+import { mockRouter, RouterMock } from '../../../../../../src/core/server/http/router/router.mock';
+import { httpServerMock } from '../../../../../../src/core/server/http/http_server.mocks';
+import { RouteValidatorConfig } from '../../../../../../src/core/server/http/router/validator';
+
+import { registerEnginesRoute } from './engines';
+import { ObjectType } from '@kbn/config-schema';
+
+jest.mock('node-fetch');
+const fetch = jest.requireActual('node-fetch');
+const { Response } = fetch;
+const fetchMock = require('node-fetch') as jest.Mocked<typeof fetch>;
```

Comment:
I had to use a `require` here instead of import type cast it to `jest.Mocked`. Couldn't figure out how to do this otherwise.

Reply:
yep, that sounds about right to me from my memory of jest mocks!